Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -304,33 +304,43 @@ public class DateRange
         /// Matches repositories with regards to the <param name="date"/>.
         /// We will use the <param name="op"/> to see what operator will be applied to the date qualifier
         /// </summary>
-        public DateRange(DateTime date, SearchQualifierOperator op)
+        [Obsolete("This ctor has been deprecated as GitHub API now supports date and time")]
+        public DateRange(DateTime date, SearchQualifierOperator op) : this(new DateTimeOffset(date), op)
+        {
+        }
+
+        /// <summary>
+        /// Matches repositories with regards to both the <param name="from"/> and <param name="to"/> dates.
+        /// </summary>
+        [Obsolete("This ctor has been deprecated as GitHub API now supports date and time")]
+        public DateRange(DateTime from, DateTime to) : this(new DateTimeOffset(from), new DateTimeOffset(to))
+        {
+        }
+
+        public DateRange(DateTimeOffset from, DateTimeOffset to)
+        {
+            query = string.Format(CultureInfo.InvariantCulture, "{0:yyyy-MM-dd'T'HH:mm:ss zzz}..{1:yyyy-MM-dd'T'HH:mm:ss zzz}", from.DateTime, to.DateTime);
```

Comment:
What will the `zzz` component be here, given you're using the `DateTime` property and not the `DateTimeOffset` itself?

Reply:
But at which offset? `DateTime` doesn't store a separate offset.

Reply:
See https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.datetime?view=netframework-4.7.2#remarks

Reply:
Hi sorry you are right @khellang. I was trying to do to code this while also doing my job which was a mistake. I'll fix in my next PR.